src/main/java/ru/sbtqa/tag/apifactory/rest/RestEntityImpl.java

```diff
@@ -68,8 +70,11 @@ public Bullet get(String url, Map<String, String> headers) throws ApiRestExcepti
             }
 
```

Comment:
Can we refact get method with performRequest?

Reply:
Kinda. We can extract work with the request itself and the headers into private functions. I can do that later.

Reply:
Ok. Do it and I'll merge it

Reply:
Can you release it right now? Later this work can be done by anyone but now the latest release just doesn't work for me.

Reply:
Actually, I've got some time right now. I'll refactor and test it.

Reply:
Ok. And accept invitation to sbtqa-team